var/spack/repos/builtin/packages/visit/package.py

```diff
@@ -415,3 +415,18 @@ def determine_version(cls, exe):
 
     # see https://github.com/visit-dav/visit/issues/20055
     unresolved_libraries = ["*"]
+
+    def test_smoke_test(self):
+        """Test visit -cli"""
+        spec = self.spec
+
+        if "~python" in spec:
+            raise SkipTest("Package must be installed with +python")
+
+        visit_cmd = Executable(spec["visit"].prefix.bin.visit)
+        visit_cmd("-cli", "-nowin", "import visit; visit.Launch(nowin=1)")
```

Comment:
```suggestion
        visit_cmd("-cli", "-nowin", "-s","{script_file.py}")
```

where script file can be something like:

```
OpenDatabase("example.silo")
AddPlot("Pseudocolor","temp")
DrawPlots()
SaveWindow()
```

We could create the script file on the fly, but need to figure out where to store example data.

Or we could create an example vtk data file on the fly (a cube with a field) and draw it.